entrypoint.sh

```diff
@@ -1,17 +1,8 @@
 #!/bin/sh
 
-set -e
+# Use the metadatastore argument supplied via cmd line or fall back to env var
+METADATASTORE_TABLE="${1:-$METADATASTORE_TABLE}"
```

Comment:
I'd name this local one differently, otherwise I think on line 8, it will always pass in the environment variable